code/modules/mining/ores_coins.dm

```diff
@@ -90,18 +91,26 @@
 		return
 	var/mob/living/carbon/human/C = hit_atom
 	if(C.head && C.head.flags_cover & HEADCOVERSEYES)
-		visible_message("<span class='danger'>[C]'s headgear blocks the sand!</span>")
+		if(cooldown < world.time - 10)

```

Comment:
we usually do cooldown = world.time + {timetocooldown}

then check it as (if(cooldown > world.time)